run_dir/design/project_samples.html

```diff
@@ -139,9 +140,8 @@ <h4>User project description
           <!--Order Info Table-->
           <table class="table table-bordered table-striped">
             <thead>
-              <tr class="darkth">
-                <th colspan="2">Order info &amp; Contacts</th>
-              </tr>
+	              <tr class="darkth"><th colspan="2">Order info &amp; Contacts <a id="psul_link_button" data-toggle="tooltip" class="btn btn-light btn-xs float-right copy_proj" target="_blank" title="Opens LIMS Dashboard and copies project name" href="https://lims-dashboard.scilifelab.se/">PSUL</a></th>
```

Comment:
Better to have the LIMS dashboard link in the settings file, so that it'll point to the dev version on genstat stage.